tox.ini

```diff
@@ -20,11 +20,11 @@ commands_pre =
 commands =
   pytest {posargs}
 
-[testenv:py3{8,9,10,11,12}-ao-prod]
-changedir = tests
-setenv =
-  SW_APM_COLLECTOR = collector.appoptics.com
-  SW_APM_SERVICE_KEY = foo-bar:service-key
+;[testenv:py3{8,9,10,11,12}-ao-prod]
```

Comment:
Nit: can also comment out L5 in this file

Reply:
Sure. Done